pylint/message_sets.py

```diff
@@ -0,0 +1,16 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/pylint-dev/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/pylint-dev/pylint/blob/main/CONTRIBUTORS.txt
+
+core: dict[str, set[str]] = {
```

Comment:
I don't love the word "core", I was just trying to avoid "all" which could be confusing as it doesn't entail `--enable-all-extensions` nor the messages "disabled by default" with the checker setting.